schemas/2016-11-01/Microsoft.DataLakeStore.json

```diff
@@ -0,0 +1,469 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2016-11-01/Microsoft.DataLakeStore.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.DataLakeStore",
+  "description": "Microsoft Data Lake Store Resource Types",
+  "resourceDefinitions": {
+    "accounts": {
+      "type": "object",
+      "description": "Microsoft.DataLakeStore/accounts",
+      "properties": {
+        "apiVersion": {
+          "enum": ["2016-11-01"]
```

Comment:
>["2016-11-01"] [](start = 18, length = 14)

can we follow the json format through the new files?

Reply:
Sorry, I do not understand.